modules/uid2IdSystem.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * This module adds uid2 ID support to the User ID module
+ * The {@link module:modules/userId} module is required.
+ * @module modules/uid2IdSystem
+ * @requires module:modules/userId
+ */
+
+import * as utils from '../src/utils.js'
+import {submodule} from '../src/hook.js';
+import {getStorageManager} from '../src/storageManager.js';
+
+const storage = getStorageManager();
+
+const MODULE_NAME = 'uid20';
```

Comment:
Can we call it `uid2` instead of `uid20`? looks inconsistent with the file name.
Also for LOG_PRE_FIX and decode function return value.

Reply:
Renamed the file.

Reply:
Cool